### PR TITLE
Added support for xpro.mit.edu subdomains on edX servers

### PR DESCRIPTION
**Pull request by blarghmatey**
#### What are the relevant tickets?
https://github.com/mitodl/mitxpro/issues/543

#### What's this PR do?
Moves the edX servers to use an xpro.mit.edu subdomain

### CODE REVIEW

**Comment by pdpinch** [comment]
Is this related to https://github.com/mitodl/mitxpro/issues/543?

**Comment by blarghmatey** [comment]
Yes, I had the wrong issue linked.

File: pillar/heroku/xpro.sls [hunk 1]
```diff
@@ -46,8 +46,8 @@
       'release_branch': 'release',
       'app_log_level': 'INFO',
       'sentry_log_level': 'WARN',
-      'logout_redirect_url': 'https://xpro.mitx.mit.edu/logout',
-      'OPENEDX_API_BASE_URL': 'https://xpro.mitx.mit.edu',
+      'logout_redirect_url': 'https://courses.xpro..edu/logout',
```
Comment (shaidar) [inline pillar/heroku/xpro.sls]: Extra dot

File: salt/orchestrate/aws/xpro_elb.sls [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+          'cms': 'studio-xpro-qa.mitx.mit.edu',
+          'lms': 'xpro-qa.mitx.mit.edu',
+          'preview': 'preview-xpro-qa.mitx.mit.edu'
+  },
+'sandbox': {
+          'cms': 'studio-xpro-qa-sandbox.mitx.mit.edu',
+          'lms': 'xpro-qa-sandbox.mitx.mit.edu',
+          'preview': 'preview-xpro-qa-sandbox.mitx.mit.edu'
+  }
+} %}
```
Comment (shaidar) [inline salt/orchestrate/aws/xpro_elb.sls]: Why aren't we using the new hostnames in this `domain_map` and if we should be, wouldn't be best if we pulled those values from `environment_settings.yml`?

**Comment by blarghmatey** [comment]
:+1:

**Pull request merged**